provision/acc_provision/flavors.yaml

```diff
@@ -81,10 +85,110 @@ flavors:
     default_version: 1.9
     status: null
     hidden: true
-    order: 15
+    order: 16
     config:
       kube_config:
         no_priority_class: True
+  openshift-4.4-openstack:
+    desc: Red Hat OpenShift Container Platform 4.4 on OpenStack
+    default_version: 5.0
+    config:
+      kube_config:
+        use_acicni_priority_class: True
+        use_external_service_ip_allocator: False
+        host_agent_openshift_resource: True
+        allow_pods_external_access: True
+        use_privileged_containers: True
+        use_openshift_security_context_constraints: False
+        use_openshift_run_level: True
+        use_cnideploy_initcontainer: True
+        allow_kube_api_default_epg: True
+        kubectl: oc
+        system_namespace: aci-containers-system
+        no_priority_class: True
+        host_agent_cni_bin_path: /var/lib
+        host_agent_cni_conf_path: /etc/kubernetes
+        generate_installer_files: True
+      aci_config:
+        disable_node_bd_creation: True
+        kube_default_provide_kube_api: True
+        items:
+          - name: openshift-svc-catalog
+            range:
+              -
+                [2379, 2379]
+            etherT: ip
+            prot: tcp
+            stateful: "no"
+            consumed: ["system"]
+            provided: ["nodes"]
+          - name: openshift-monitoring
+            range:
+              -
+                [9091, 9091]
+              -
+                [9094, 9094]
+              -   
+                [9095, 9095]
+              -   
+                [3000, 3000]
+              -   
+                [8080, 8080]
+            etherT: ip
+            prot: tcp
+            stateful: "no"
+            provided: ["system","default","nodes"]
+            consumed: ["system","default"]
+        kube_api_entries:
+          - name: etcd1
+            range: [2379, 2379]
+            etherT: ip
+            prot: tcp
+            stateful: "no"
```

Comment:
I think we need to add 2380 here (so perhaps a range 2379-2380). Can you double check with abhishek?

Reply:
@abhis2112 can you please confirm?

Reply:
@apoorva11029 yes, now we need for 2379, 2380, 9979, 9978
So the new ports are 2380 and 9978

Reply:
Added, thanks!